www/mod_ca/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                mod_ca
+version             0.2.2
+categories          www security
+platforms           darwin
+maintainers         redwax.org:dirkx @dirkx openmaintainer
```

Comment:
this needs curly brackets to signify they belong to the same person, so like:
```{redwax.org:dirkx @dirkx} openmaintainer```

Reply:
will do.